test/common.js

```diff
@@ -107,6 +107,26 @@ Object.defineProperty(exports, 'localhostIPv4', {
   }
 });
 
+Object.defineProperty(exports, 'broadcastIPv4', {
+  if (broadcastIPv4 !== null) return broadcastIPv4;
```

Comment:
Need to add a `var broadcastIPv4`, probably around line 71 (right below the lone for `localhostIPv4`), no?